lib/order.rb

```diff
@@ -0,0 +1,39 @@
+require_relative 'menu'
+
+class Order
+  attr_reader :sum, :basket, :menu
+
+  def initialize(order_menu)
+    @menu = order_menu.list
+    @basket = Hash.new(0)
+    @sum = 0
```

Comment:
Can this be calculated every time it's asked for, rather than stored on the order object?